python/pyspark/sql/tests/connect/test_connect_basic.py

```diff
@@ -226,6 +226,17 @@ def test_error_handling(self):
         with self.assertRaises(AnalysisException):
             df.collect()
 
+    def test_error_stack_trace(self):
+        self.connect.conf.set("spark.sql.pyspark.jvmStacktrace.enabled", "true")
+        with self.assertRaises(AnalysisException) as e:
+            self.connect.sql("select x").collect()
+        self.assertTrue("JVM stacktrace" in e.exception.message)
```

Comment:
Can we match the true AnalysisException error messages? It should be something like `column_not_resolved`?